rviz_common/src/rviz_common/visualizer_app.cpp

```diff
@@ -307,6 +309,7 @@ bool VisualizerApp::init(int argc, char ** argv)
 VisualizerApp::~VisualizerApp()
 {
   delete continue_timer_;
+  rclcpp::shutdown();
```

Comment:
This needs to be wrapped into the `ros_integration` namespace, see how init and ok are done and emulate that.